wagtail/admin/templates/wagtailadmin/pages/index.html

```diff
@@ -86,7 +86,11 @@
 
                         // Post
                         $.post(url, {csrfmiddlewaretoken: CSRFToken}, function(){
-                            addMessage('success', '"' + $(movedElement).data('page-title') + '" has been moved successfully.')
+                            document.dispatchEvent(
+                                new CustomEvent('w-messages:add', {
+                                    detail: { status:'success', text:'"' + $(movedElement).data('page-title') + '" has been moved successfully.' },
+                                }),
+                            );
```

Comment:
```suggestion
                            const text = `"${(movedElement).data('page-title')}" has been moved successfully.`;
                            const event = new CustomEvent('w-messages:add', { detail: { status: 'success', text } });
                            document.dispatchEvent(event);
```

A few recommendations here

* Let's build the string here with template literals, it's easier to read as above and we no longer need to support IE11 (hence why the old style was here). https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals
* This is a bit tricky to read with the multiple nesting, setting a named variable instead for the text and the event on their own lines makes this much readable. It's still 3 lines but much easier to follow what's happening.
* Optional: If our event default is `success`, we could omit the `status: 'success'`, assuming we make this message type the default in the controller. This may make the code a bit harder to follow long term but as these are the only two usages of this event, maybe it's ok for this refactor.